docs/specs/app-specs.md

```diff
@@ -1,121 +1,186 @@
-# Dusty App Specs
-These specs are used to specify apps for you local environment. They should go in the apps folder in your specs repo.
+# App Specs
 
-## Keys
+Apps define running containers based on source code under your control. App specs tell
+Dusty how to run this container, including any dependencies the app may have on other
+apps, services, or libs. You can also specify common scripts and testing commands for
+an app in its spec.
+
+By default, Dusty manages the source for an app for you and keeps it up to date automatically.
+You can tell Dusty to use your own locally checked out copy of the source using `dusty repos`.
```

Comment:
I think it's more important that docs show people what they're looking for than that they're DRY. If we were using a more robust docs system we could have fact tables and stuff but we don't, do we just end up copy-pasting.

Do you not think this is useful here to define what an app is?